jinseok/service/postService.js

```diff
@@ -0,0 +1,29 @@
+const postDao = require('../model/postDao');
+
+const createPost = async (context, userId) => {
+  return await postDao.createPost(context, userId);
```

Comment:
```suggestion
  return postDao.createPost(context, userId);
```
굳이 따지자면 비동기처리가 없어도 되는 코드입니다 :)